aws_codeseeder/services/cloudwatch.py

```diff
@@ -53,13 +53,18 @@ def get_stream_name_by_prefix(
         Name of the CloudWatch Logs Stream (if found)
     """
     client = boto3_client("logs", session=session)
-    response = client.describe_log_streams(
+
+    response = try_it(
```

Comment:
where is this defined?  I don't see it in the PR

Reply:
L20 imported from utils

Reply:
cool!  didn't know it existed

Reply:
Yep we're using it for other service calls (codebuild) but not for logs yet